README.md

```diff
@@ -75,6 +75,24 @@ To set up a development environment for the CLI
 6. Start the in-cluster API replacement job (powered by [http://tilt.dev](tilt.dev)) with `make cluster-dev`
 7. make or make unit-tests to ensure everything built correctly.
 
+### Pre-commit hooks
+
+This repo has [pre-commit hooks](https://pre-commit.com/) configured on it, they are run by github action for pull requests but it is recommended install them locally too:
+
+```bash
+$ pip install pre-commit
```

Comment:
I would prefer not to involve Python in our build process. I am not familiar enough with it or it's ecosystem to know how to troubleshoot it.

Reply:
Is the problem here using a program written in python or using pip to install it? pre-commit can also be [installed](https://pre-commit.com/#install) via brew on conda (although I'm not familiar with conda myself) and I can suggest one of those, or just link to the relevant documentation.

Reply:
It’s really about the pip command itself. I haven’t had good luck with it.

Reply:
I've updated the readme with details on installation via brew and conda. Are those better options?

Reply:
I am on linux, and not familiar with conda.

Reply:
yea I'd not encountered conda before.

I looks like some linux distros have pre-commit available in their package managers (e.g. ubuntu jammy) but I suspect it's only in very recent ones.

Is this a blocker?

Reply:
Yes I would prefer we not involve a new runtime in the build if possible.

Reply:
Is there a go version?

Reply:
Not that I am aware of